images/sqlx-cli-with-migrations/Dockerfile

```diff
@@ -5,4 +5,17 @@ LABEL org.opencontainers.image.source https://github.com/kamu-data/kamu-cli
 
 ################################################################################
 
+# Install dependencies
+RUN apk --no-cache add \
+    bash \
+    curl \
+    jq && \
+    # Download and install the AWS CLI version 2
+    curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip" && \
+    unzip awscliv2.zip && \
+    ./aws/install && \
+    rm -rf awscliv2.zip aws
+
+COPY scripts /scripts
```

Comment:
1) We need to add execute permissions for scripts:

```
COPY --chmod=0755 ...
```

2) Move them to a path where they will add in `PATH`

Current value:
```
PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
```

Reply:
added permissions, don't see need in PATH